src/notebooks/outputs/tracebackFormatter.ts

```diff
@@ -23,7 +23,7 @@ export class NotebookTracebackFormatter implements ITracebackFormatter {
         return traceback.map((traceFrame) => this.modifyTracebackFrameIPython(cell, traceFrame));
     }
     private modifyTracebackFrameIPython(cell: NotebookCell, traceFrame: string): string {
-        if (/^[Input|File].*?\n.*/.test(traceFrame)) {
+        if (/^[Cell|Input|File].*?\n.*/.test(traceFrame)) {
```

Comment:
It looks like this code path also need the Cell regex. Or was the intention to just fix for IW first?

Reply:
adding cell here still gets into the more generic formatting that we do, just skips getting the cell URI